docs/api.md

```diff
@@ -748,8 +748,15 @@ const [response] = await Promise.all([
 Shortcut for [page.mainFrame().click(selector[, options])](#frameclickselector-options).
 
 #### page.close()
+- `options` <[Object]>
+  - `runBeforeUnload` <[boolean]> Defaults to `false`. Whether to run the before unload page handlers.
```

Comment:
I feel like this is a bit confusing, because before Unload handlers might still not run if there was no user interaction.

Reply:
Can we get a link to mdn on beforeunload here?

Reply:
> I feel like this is a bit confusing, because before Unload handlers might still not run if there was no user interaction.

I agree. However, there are many different heuristics in place, and they keep changing. I don't think that has anything to do with pptr or our automation API - that's how WebPlatform rolls.

> Can we get a link to mdn on beforeunload here?

Done.